app/controllers/categories_controller.rb

```diff
@@ -1,2 +1,24 @@
 class CategoriesController < ApplicationController
+
+	def index
+		@categories = Category.where("question_id = '#{params[:question_id]}'")
+	end
+
+	def new
+		@category = Category.new
+		@question = Question.find(params[:question_id])
+	end
+
+	def create
+		params[:category][:question_id] = params[:question_id]
+		category = Category.create(category_params)
+		redirect_to question_categories_path
```

Comment:
dafuq?  I think this is similar to my previous comment.  Why not:

`Question.find(...).categories.create()` or some such.

Look up the rails guide on associations